setup.py

```diff
@@ -20,10 +23,24 @@ def get_long_desc():
     desc += open('CHANGES').read()
     return desc
 
+# foolproof way of single-sourcing version as per
+# http://stackoverflow.com/a/24517154/112943
```

Comment:
@TimDumol : Thinking about it more, I think the link is worth keeping, but I'd add something about why it isn't done in exactly he same way. Perhaps:

```python
# Foolproof way of single-sourcing version inspired
# by http://stackoverflow.com/a/24517154/112943
# Here we use re.search instead of exec to avoid any
# possibility of side effects in version.py
```

It's probably not super-important, I've just been thinking about code commenting a lot lately. I prefer documenting the reason for doing something in addition to where the idea came from.